public/js/config-i18n.js

```diff
@@ -0,0 +1,19 @@
+// intialize banana-i18n constructor
+const banana = new Banana({ finalFallback: 'en' });
+// set locale
+banana.setLocale(window.locale);
+// setup fetch api with async/await to load json files
+async function fetchJson() {
```

Comment:
Can we fetch the locales directly instead of making a http call ?

Reply:
Thats seems the drawback of the library they only provide ```banana.load()``` which can hold the constant having messages defined inside it . for loading messages from json explicitly they suggest fetch()

here is the documentation - https://github.com/wikimedia/banana-i18n#loading-the-messages
there demo example uses the same  - https://github.com/wikimedia/banana-i18n/blob/master/demo/js/demo.js

Current library is a model of``` jquery.i18n ```and purely written in js .  Jquery I18n provides load method to load translations but not this one
https://github.com/wikimedia/jquery.i18n#dynamic-loading-using-load-method

Reply:
You can load the i18n translations to messages object and pass it to the constructor. It is just that banana is doesn't dictate a directory structure. You need to implement loading yourself

Reply:
Yes, loading is to be done manually, so gone with fetch API 

Now I am thinking of bringing translations in form of J's objects and parse them using json.parse for  further use

Any suitable loading technique you want to suggest here? As there can be more than one techniques?

Reply:
Using fetch require http call, and it won't play nicely with the offline app. Here is one approach with load method https://github.com/zhaofengli/refill/blob/2ee4c00623f20bbf9b86e76679f763217f7a241e/web/src/main.js#L36

Reply:
Another approach https://github.com/smc/mlmorph-web/blob/5d741f617c0ed541fb65a249b039d74d1daf4d15/src/main.js#L9